src/psyclone/tests/domain/lfric/lfric_builtins_test.py

```diff
@@ -3305,24 +3555,26 @@ def test_setval_X(tmpdir, monkeypatch, annexed, dist_mem, fortran_writer):
             output_dm_2 = output_dm_2.replace("dof_annexed", "dof_owned")
         assert output_dm_2 in code
 
-
 # ------------- Inner product of real fields -------------------------------- #
 
 
 def test_X_innerproduct_Y(tmpdir, dist_mem):
-    ''' Test that 1) the str method of LFRicXInnerproductYKern returns the
-    expected string and 2) we generate correct code for the built-in
-    operation which calculates inner product of real-valued fields X and Y
-    as innprod = innprod + X(:)*Y(:).
+    '''Test that 1) the '__str__' method of 'LFRicXInnerproductYKern'
+    returns the expected string and 2) we generate correct code for
+    the built-in operation which calculates inner product of
+    real-valued fields 'X' and 'Y' as 'innprod = innprod +
+    X(:)*Y(:). 3)' Also test the 'metadata()' method.
```

Comment:
Quotation mark should be after `Y(:)` and not after `3)`

Reply:
Oops, I was obviously going too fast and not checking enough. Sorry.